src/gui/about_imp.h

```diff
@@ -36,6 +36,11 @@
 #include "base/utils/misc.h"
 #include "base/unicodestrings.h"
 
+#define SIXTYFOUR ""
```

Comment:
I either don't mark it or mark it explicitly, because a user not familiar with the version semantic won't able to deduce that no "64 bit" markings implies 32 bit.

Talking about usual apps, my x64 firefox has no "64 bit" stated in the about dialog. I guess it doesn't really matter to most users and they shouldn't have concern for it.